src/analysis/history.cpp

```diff
@@ -235,4 +236,46 @@ Real ReduceMagneticFluxPhi(MeshData<Real> *md) {
   return 0.5 * result; // 0.5 \int detg B^r dx2 dx3
 } // Phi
 
+// SN analysis
+
+void ReduceLocalizationFunction(MeshData<Real> *md) {
```

Comment:
Maybe add a comment above this indicating it's not currently in use?

Reply:
done